lib/matplotlib/__init__.py

```diff
@@ -660,12 +661,36 @@ class RcParams(MutableMapping, dict):
     --------
     :ref:`customizing-with-matplotlibrc-files`
     """
-
     validate = rcsetup._validators
+    namespaces = ("backends", "lines", "patches", "hatches", "boxplot", "font", "text",
```

Comment:
I'm very paranoid of not making more API public than absolutely necessary.
```suggestion
    _namespaces = ("backends", "lines", "patches", "hatches", "boxplot", "font", "text",
```

Reply:
Fair point. I'll make sure all unnecessary (API) variables are private